src/Domain/CoordinateMaps/CoordinateMap.hpp

```diff
@@ -467,11 +468,33 @@ CoordinateMap<SourceFrame, TargetFrame, Maps...>::inv_jacobian_impl(
               return nullptr;
             });
 
+        auto map_overload = make_overloader(
```

Comment:
It looks like this overloader occurs twice.  Can it be factored out into a struct or overloaded functions?